link/uprobe.go

```diff
@@ -254,11 +262,12 @@ func (ex *Executable) uprobe(symbol string, prog *ebpf.Program, opts *UprobeOpti
 
 	offset := opts.Offset
 	if offset == 0 {
+		offset = opts.RelativeOffset
```

Comment:
nit: what about doing `offset: offset+opts.RelativeOffset` in `probeArgs` on line 287? this way the "relative offset" is added both for a specified one and for an internally calculated one

Reply:
@mmat11 done with https://github.com/mertyildiran/ebpf/commit/8a7955ac654c14d0744dc3e48185802c84d94944